django/db/models/indexes.py

```diff
@@ -14,6 +14,9 @@ class Index(object):
     suffix = 'idx'
 
     def __init__(self, fields=[], name=None):
+        # Sanity checks for fields argument
+        if type(fields) != list:
```

Comment:
Considering "most places in Django accept either" I don't see any strong reason for not doing that here. Thanks.